index.bs

```diff
@@ -109,20 +202,41 @@ result or an error being returned to the [=local end=]. Multiple
 commands can run at the same time, and commands can potentially be
 long-running. As a consequence, commands can finish out-of-order.
 
-Each concrete [=command=] type is defined by:
- - A string <dfn export for=command>name</dfn>, implicit in its
-   definition.
- - A <dfn export for=command>parameter type</dfn>, which is a |map|
-   whose entries are the inputs to the command.
- - A set of [=remote end steps=].
- - A <dfn export for=command>return type</dfn>, which may be null.
+Each [=command=] is defined by:
+- A <dfn export for=command>command id</dfn>. This is an integer id
+  provided by the [=local end=] for each command. The value of the id is not
```

Comment:
Thinking some more about this, I'm not sure the command ID should be mentioned here at all. It's a transport-layer concept that has no bearing on the definition of the command itself. Maybe the definition and requirements for the command ID should go in the transport section instead?

Reply:
The requirements are already there. I've changed things so that this section is only stating facts about the command id; I couldn't see anywhere in the transport section that would make it clearer.

Reply:
Thanks. The new description is more clear.